OperationDefinition/OperationDefinition-genomics-generate.json

```diff
@@ -0,0 +1,75 @@
+{
+  "resourceType": "OperationDefinition",
+  "id": "OperationDefinition-genomics-generate",
+  "url": "https://fhir.nhs.uk/OperationDefinition/genomics-generate",
+  "version": "0.0.1",
+  "name": "GenomicsGenerate",
+  "status": "draft",
+  "kind": "operation",
+  "experimental": false,
+  "date": "2024-09-04",
+  "publisher": "NHS England",
+  "description": "Used to generate a DocumentReference resource for a genomic data file, optionally pulling the referenced data file to a central data store and generating a DRS url for the data file.\r\nCustomised backport of the R5 DocumentReference-generate OperationDefinition: http://hl7.org/fhir/OperationDefinition/DocumentReference-generate.\r\nSpecific profiles for input or output parameters, e.g. for certain data file types, are pending further investigation by GEL.",
```

Comment:
description: May be start with: An operation used generate a DocumentReference resource for..............................

Reply:
Added